.github/workflows/linux.yml

```diff
@@ -674,8 +677,9 @@ jobs:
 
           source ${{ env.INSTALL_DIR }}/setupvars.sh
 
-          python3 -m pytest ${{ env.MODEL_HUB_TESTS_INSTALL_DIR }}/tf_hub_tests/ -m precommit --junitxml=${{ env.INSTALL_TEST_DIR }}/TEST-tf_hub_tf_fe.xml
+          python3 -m pytest ${{ env.MODEL_HUB_TESTS_INSTALL_DIR }}/tf_hub_tests/ -m ${{ env.TYPE }} --junitxml=${{ env.INSTALL_TEST_DIR }}/TEST-tf_hub_tf_fe.xml --html TEST-tf_hub_tf_fe.html --self-contained-html
         env:
+          TYPE: ${{ github.event_name == 'schedule' && 'nightly' || 'precommit'}}
```

Comment:
How long does it take to execute a full nightly scope?

Reply:
Discussed offline